### PR TITLE
Swolfson project-react-native-app

### PR DESCRIPTION
**Pull request by swolfson1**
in this weeks project i created a simple todo list. the app was functional as of few days ago before i changed the styling to styled components. I have spent hours and hours trying to figure out why its not working.  I am confident that my JS looks good and functions as it should, but the rest is just not working i need help getting it run again.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,26 +1,79 @@
...
+
+  const pressHandler = (key) => {
+    setTodos(prevTodos => {
+      return prevTodos.filter(todo => todo.key != key);
+    });
+  };
+
+  const submitHandler = (text) => {
+    if (text.length > 5) {
+      setText('');
```
Comment (JennieDalgren) [inline App.js]: you don't have access to `setText` here in this file. You need to do this reset in the AddTodo component where that state lives.

File: App.js [hunk 2]
```diff
@@ -1,26 +1,79 @@
...
+      setTodos(prevTodos => {
+        return [
+          { text, key: Math.random().toString() },
+          ...prevTodos
+        ];
+      });
+    } else {
+      Alert.alert('OOPS', 'New todo is to short, try again', [
+        { text: 'Understood', onPress: () => console.log('closed') }
+      ]);
```
Comment (JennieDalgren) [inline App.js]: good error management here

File: App.js [hunk 3]
```diff
@@ -1,26 +1,79 @@
...
-  justify-content: center;
-  align-items: center;
+  
 `
-
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
+const Content = styled.View`
+paddingTop: 20;
```
Comment (JennieDalgren) [inline App.js]: this is styled components, therefor you should use normal css syntax in here.

`padding-top: 20;`

File: App.js [hunk 4]
```diff
@@ -1,26 +1,79 @@
...
+  
+`
+const List = styled.View`
+marginTop: 20;
```
Comment (JennieDalgren) [inline App.js]: this is styled components, therefor you should use normal css syntax in here.

File: components/Header.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    <HeaderTop>
+      <Title></Title>
+    </HeaderTop >
+  );
+}
+
+const HeaderTop = styled.View`
+height: 80;
+paddingTop: 38;
+backgroundColor: coral;
```
Comment (JennieDalgren) [inline components/Header.js]: this is styled components, therefor you should use normal css syntax in here.

File: components/Header.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  
+`
+
+const Title = styled.Text`
+textAlign: center;
+color: #fff;
+fontSize: 20;
+fontWeight: bold;
```
Comment (JennieDalgren) [inline components/Header.js]: this is styled components, therefor you should use normal css syntax in here.

File: components/TodoItem.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { TouchableOpacity, Text } from 'react-native';
+import styled from "styled-components"
+
+
+const TodoItem = ({ pressHandeler, item }) => {
```
Comment (JennieDalgren) [inline components/TodoItem.js]: typo here, `pressHandler` not pressHandeler

File: components/TodoItem.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+
+const Item = styled.Text`
+  paddingTop: 16;
+   marginTop: 16;
+    borderColor: #bbb;
+    borderWidth: 1;
+    borderStyle: "dashed";
+    borderRadius: 10;
+
```
Comment (JennieDalgren) [inline components/TodoItem.js]: this is styled components, therefor you should use normal css syntax in here.

File: components/addTodo.js [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    </View>
+  );
+}
+
+const Input = styled.TextInput`
+  marginBottom: 10px;
+    paddingHorizontal: 8px;
+    paddingVertical: 6px;
+    borderBottomWidth: 1px;
+    borderBottomColor: #ddd;
```
Comment (JennieDalgren) [inline components/addTodo.js]: this is styled components, therefor you should use normal css syntax in here.

**Comment by JennieDalgren** [comment]
@swolfson1  could you push your latest changes here so I can see that the app is working?

**Comment by swolfson1** [comment]
Hey jennie! I made the changes it should work now:)

**Pull request closed without merging**